### PR TITLE
Foundation for new styles, fonts and themes; updated connected wallet functionality links

### PR DESCRIPTION
**Pull request by VadimSaveljev**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -16,18 +16,21 @@ import Pools from '@pages/Assets/Pools/Pools';
 import Tokens from '@pages/Assets/Tokens/Tokens';
 import Discover from '@pages/Discover/Discover';
 import MyAssets from '@pages/MyAssets/MyAssets';
+import NewNftMint from '@pages/MyAssets/NftMint/NewNftMint';
```
Comment (jsidorenko) [inline src/App.tsx:19]: Could it be named `NftMint` instead of `NewNftMint`?

Comment (jsidorenko) [reply src/App.tsx:19]: a, wait, I see the `NftMint` page already exists

File: src/App.tsx [hunk 2]
```diff
@@ -16,18 +16,21 @@ import Pools from '@pages/Assets/Pools/Pools';
...
+import MyPools from '@pages/MyAssets/Pools/MyPools';
 import CollectionEdit from '@pages/Nfts/Collections/CollectionEdit';
 import CollectionMint from '@pages/Nfts/Collections/CollectionMint';
 import Collections from '@pages/Nfts/Collections/Collections';
 import NftEdit from '@pages/Nfts/Nfts/NftEdit';
 import NftMint from '@pages/Nfts/Nfts/NftMint';
 import Nfts from '@pages/Nfts/Nfts/Nfts';
 import Swap from '@pages/Swap/Swap';
+import PoolCreate from '@pages/MyAssets/Pools/PoolCreate';
```
Comment (jsidorenko) [inline src/App.tsx]: we need to check the sorting rules for imports. I think I should go under the `'@pages/MyAssets/..`

File: src/App.tsx [hunk 3]
```diff
@@ -77,6 +80,18 @@ const App = () => {
                     </PrivateRoute>
                   }
                 />
+
+                <Route path={routes.myAssets.newNftMint} element={<NewNftMint />}>
+                  <Route
+                    path={routes.myAssets.selectCollection}
+                    element={
+                      <PrivateRoute>
+                        <div>za za</div>
```
Comment (jsidorenko) [inline src/App.tsx]: ?

File: src/App.tsx (hunk 1 above)
Comment (VadimSaveljev) [reply src/App.tsx:19]: check https://github.com/SaltyCucumber/assets-portal/pull/55/files#diff-96453fc4941066bdda4cac7e2cc6e541a9d18657f1f03499f36dc6f47e3cd5bd

I've layed ground for the new flow, will remove/rename old one once that is done

File: src/components/Common/Loader.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { useModalStatus } from '@contexts/ModalStatusContext';
 
 import { ModalStatusTypes } from '@helpers/constants';
 
-import PolkadotIcon from '@images/polkadot-icon.svg';
+import PolkadotIcon from '@images/icons/polkadot.svg';
```
Comment (jsidorenko) [inline src/components/Common/Loader.tsx:8]: 👍

File: src/App.tsx (hunk 2 above)
Comment (VadimSaveljev) [reply src/App.tsx]: haven't run format, will do

File: src/components/Common/Title.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { memo } from "react";
+import styled from "styled-components"
```
Comment (jsidorenko) [inline src/components/Common/Title.tsx]: A semicolon is missed. P.s. why prettier didn't auto-fix that?

Comment (VadimSaveljev) [reply src/components/Common/Title.tsx]: I didn't run format

**Review by jsidorenko — APPROVED** [review]

**Pull request merged**